openassessment/xblock/openassessmentblock.py

```diff
@@ -116,10 +117,10 @@ class OpenAssessmentBlock(
         help="ISO-8601 formatted string representing the submission due date."
     )
 
-    allow_file_upload = Boolean(
-        default=False,
+    file_upload_type = String(
+        default=None,
         scope=Scope.content,
-        help="File upload allowed with submission."
+        help="File upload to be included with submission (can be 'image' or 'file')."
```

Comment:
why the change from "allowed" to "to be included"?

Also I think that parenthetical would benefit from a broader description - for example, "(can be None, 'image', to allow .jpg files, or 'file', to allow .pdf files)"  --- basically, to clarify what all of those options mean.